### PR TITLE
Throw exception if a tf.keras model does not set TF_KERAS

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/main.py [hunk 1]
```diff
@@ -75,6 +75,14 @@ def convert_keras(model, name=None, doc_string='', target_opset=None, channel_fi
     set_logger_level(logging.DEBUG if debug_mode else logging.INFO)
     tf2onnx.logging.set_level(logging.DEBUG if debug_mode else logging.INFO)
 
+    if isinstance(model, tf.keras.Model):
+        if not is_tf_keras:
+            k2o_logger().warning("WARNING: This is a tensorflow keras model, but keras standalone converter is used." +
+                                 " Please set environment variable TF_KERAS.")
```
Comment (vinitra) [inline keras2onnx/main.py]: Specify TF_KERAS=1

File: keras2onnx/main.py [hunk 2]
```diff
@@ -75,6 +75,14 @@ def convert_keras(model, name=None, doc_string='', target_opset=None, channel_fi
...
+    elif is_tf_keras:
+        k2o_logger().warning("WARNING: This is a keras standalone model, but tensorflow.keras converter is used." +
+                             " Please unset environment variable TF_KERAS.")
```
Comment (vinitra) [inline keras2onnx/main.py]: Specify how to unset -- TF_KERAS=0?

Comment (jiafatom) [reply keras2onnx/main.py]: set TF_KERAS=

File: keras2onnx/main.py (hunk 1 above)
Comment (jiafatom) [reply keras2onnx/main.py]: As long as the environment variable is set, then it will use tf.keras.

File: keras2onnx/main.py [hunk 3]
```diff
@@ -75,6 +75,14 @@ def convert_keras(model, name=None, doc_string='', target_opset=None, channel_fi
     set_logger_level(logging.DEBUG if debug_mode else logging.INFO)
     tf2onnx.logging.set_level(logging.DEBUG if debug_mode else logging.INFO)
 
+    if isinstance(model, tf.keras.Model):
+        if not is_tf_keras:
+            k2o_logger().warning("WARNING: This is a tensorflow keras model, but keras standalone converter is used." +
+                                 " Please set environment variable TF_KERAS.")
+    elif is_tf_keras:
```
Comment (wenbingl) [inline keras2onnx/main.py]: This never gonna happen.

File: keras2onnx/main.py (hunk 1 above)
Comment (wenbingl) [inline keras2onnx/main.py]: raise an exception since it can't proceed any further.

File: keras2onnx/main.py (hunk 3 above)
Comment (jiafatom) [reply keras2onnx/main.py]: It happens when this is a keras standalone model

**Pull request merged**